ci-operator/step-registry/openshift/e2e/test/network-qe/openshift-e2e-test-network-qe-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: openshift-e2e-test-network-qe
+  env:
+  - name: TEST_SCENARIOS
+    default: SDN
+  - name: E2E_RUN_TAGS
+    default: FDP
+  - name: TEST_TIMEOUT
+    default: "35"
+  steps:
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: cucushift-e2e
+    env:
+    - name: E2E_RUN_TAGS
+      default: "FDP"
```

Comment:
The env here is not needed.

Reply:
@liangxia You mean this whole block is not needed?
```
env:
    - name: E2E_RUN_TAGS
      default: "FDP"
```

Cucushift would honor go TEST_FILTERS?

Reply:
Ah right, E2E_RUN_TAGS for cucushift will use format @fdp .